boards/arm/nucleo_l496zg/Kconfig.board

```diff
@@ -1,6 +1,6 @@
 # Kconfig - STM32L496ZG Nucleo board configuration
 #
-# Copyright (c) 2018 Ioannis Konstantelias <ikonstadel@gmail.com>
```

Comment:
Please fix in initial commit rather than adding a fixing commit

Reply:
Done.